packages/language-server/src/completion-items.ts

```diff
@@ -252,6 +257,43 @@ function createTextEditsForClassInTagName(
   return { range, newText, filterText };
 }
 
+function addClosingTagTextEdit(
+  xmlElement: XMLElement,
+  closingTagName: string,
+  additionalTextEdits: TextEdit[]
```

Comment:
Please avoid **mutating** `additionalTextEdits` in this function.
instead return an array of additional textEdits and concat it **outside** this function.

This is a more functional style like most of our other code here and will make it easier to follow the logical flow.

Reply:
Done